### DISCUSSION TITLE
How to test Broadcast and Multicast

### DISCUSSION THREAD

**Discussion by luizsoares87** [esnet/iperf#1458] [General]
Hello everyone,

I'm newer using this tool and I would to test and stress the network over broadcast and multicast packets.

Someone knows if is possible to do that ?

Kind regards,

Luiz Soares

**Comment by bmah888** [comment]
Basically no. iperf3 was designed to test performance between two hosts (one acting as a client and one acting as a server). iperf2 has some functionality for doing tests with broadcast and multicast traffic.

Bruce.
